app/(main)/admin/components/SongsTable.tsx

```
@@ -0,0 +1,191 @@
+          <TextButton 
+          className='bg-[--md-sys-color-surface-container-high] rounded-md'
+          onClick={() => {
+            queryClient.invalidateQueries({ queryKey: ['songs'] })
+            refetch()
+            }}>
+            Refresh
+          </TextButton>
+          <button
+            onClick={() => setIsAddModalOpen(true)}
+            className="px-4 py-2 bg-[--md-sys-color-primary] text-[--md-sys-color-on-primary] rounded-md"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)